pytest_html/plugin.py

```diff
@@ -197,6 +197,8 @@ def append_extra_html(self, extra, extra_index, test_index):
                                              extra.get('extension'))
 
             elif extra.get('format') == extras.FORMAT_TEXT:
+                if isinstance(extra.get('content'), bytes):
+                    extra['content'] = extra['content'].decode("utf-8")
```

Comment:
Yes, I will move it. In current implementation if content of extra text contains bytes string with utf-8 encoded chars it will raise UnicodeDecodeError. So this fix will always decode bytes strings for extra text.
For example: `"\xe2\x80\xb1" -> u"\2031" `